assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,145 @@
-console.log('***** Music Collection *****')
+console.log('***** Music Collection *****');
+
+const collection = [];
+
+const addToCollection = (title, artist, yearPublished, tracks) => {
+  const album = {
+    title,
+    artist,
+    yearPublished,
+    tracks,
+  };
+  collection.push(album);
+  return album;
+};
+
+const showCollection = (collectionArray) => {
+  console.log(collectionArray.length);
+  collectionArray.forEach((album) => {
```

Comment:
Oh `.forEach`. What purpose do you serve now that for...of exists? I don't know, but some style guides disagree with me for some reason.